lib/pacemaker/libpacemaker_private.h

```diff
@@ -715,6 +715,11 @@ uint32_t pcmk__instance_update_ordered_actions(pe_action_t *first,
                                                uint32_t type,
                                                pe_working_set_t *data_set);
 
+G_GNUC_INTERNAL
+enum pe_action_flags pcmk__collective_action_flags(pe_action_t *action,
```

Comment:
nah just name it `pcmk__clone_action_flags` and toss in a `#define pcmk__bundle_action_flags pcmk__clone_action_flags`

... Yeah "collective" is good, I can't come up with anything besides "nonsequential" to differentiate it from groups either. I support this.